### PR TITLE
Fix chunkator when the primary key is a OneToOneField

### PR DESCRIPTION
**Pull request by zebuline**
Refs #19

### CODE REVIEW

File: demo/demo_chunkator/tests.py [hunk 1]
```diff
@@ -100,6 +89,78 @@ def test_chunk_uuid(self):
...
+        )
+        Cover.objects.create(book=book, code='player')
+        book = Book.objects.create(
+            title="Guards! Guards!",
+            author="Pratchett, Terry"
+        )
+        Cover.objects.create(book=book, code='guards')
+
+        user = User.objects.create(name='Wonder Woman')
+        Profile.objects.create(user=user, avatar='wonderfull')
```
Comment (brunobord) [inline demo/demo_chunkator/tests.py]: wonderful

File: chunkator/__init__.py [hunk 2]
```diff
@@ -28,7 +36,18 @@ def chunkator(source_qs, chunk_size, query_log=None):
         if "pk" not in source_qs._fields:
             raise MissingPkFieldException("The values() call must include the `pk` field")  # noqa
 
-    source_qs = source_qs.order_by('pk')
+    if order_by_field != 'pk':
+        field = source_qs.model._meta.get_field(order_by_field)
+        if not field.primary_key:
```
Comment (brunobord) [inline chunkator/__init__.py]: works only for primary keys... I'm fine with it, but I'd rather also allow if the field has ``unique=True`` ; not all unique indexes are primary keys.

**Review by brunobord — CHANGES_REQUESTED** [review]
small changes, it may not be possible to enlarge the scope to unique fields.

File: chunkator/__init__.py (hunk 2 above)
Comment (zebuline) [reply chunkator/__init__.py]: may be later ? :)

Comment (zebuline) [reply chunkator/__init__.py]: As discussed IRL: it is possible to have a field with `unique=True` and `null=True`. So we chosed not to support unique fields which are not primary keys.

**Comment by zebuline** [comment]
@brunobord OPM ?

**Comment by zebuline** [comment]
Don't move ! I simplified the PR :)

File: chunkator/__init__.py [hunk 3]
```diff
@@ -12,6 +12,14 @@ class MissingPkFieldException(Exception):
     pass
 
 
+class NoPkException(Exception):
```
Comment (brunobord) [inline chunkator/__init__.py]: what has happened to this exception? looks like it's not used anymore... :/

**Review by brunobord — CHANGES_REQUESTED** [review]
didn't you forget to clean something up?

File: chunkator/__init__.py (hunk 3 above)
Comment (zebuline) [reply chunkator/__init__.py]: oops, thx :)

**Review by brunobord — APPROVED** [review]

**Comment by brunobord** [comment]
now you can rebase #21

**Comment by zebuline** [comment]
\o/

**Pull request merged**